Wikipedia/Code/UILabel+WMFAuth.swift

```diff
@@ -0,0 +1,13 @@
+
+extension UILabel {
+    /// Makes an attributed string from two strings performing a **$1** substitution and applying a common color scheme (black/blue) used by a few auth view controllers.
+    /// Also re-uses the label's font so the resulting attributed string respects storyboard dynamic type font choice and responds to dynamic type size changes.
+    ///
+    /// - Parameters:
+    ///   - dollarSignString: A localization string which has a **$1** subtitution for the string from the **substitutionString** parameter.
+    ///   - substitutionString: The string which will replace **$1** in the **dollarSignString** parameter.
+    /// - Returns: Attributed string from the two parameter strings, with the **dollarSignString** in black and the **substitutionString** in blue.
+    func wmf_authAttributedStringReusingFont(withDollarSignString dollarSignString: String, substitutionString: String) -> NSAttributedString {
```

Comment:
Oh and it's a UILabel extension so it can re-use the label's _font_ so the dynamic type auto-adjustment still happens with the attributed string...

[NSForegroundColorAttributeName : UIColor.black, NSFontAttributeName : **_font_**]

^ hmm maybe in this case I should have left "self" for clarity... **_self.font_**...